python/kfserving/kfserver/storage.py

```diff
@@ -0,0 +1,44 @@
+import os
+import shutil
+import errno
+import logging
+
+_GCS_PREFIX = "gs://"
+_S3_PREFIX = "s3://"
+_LOCAL_PREFIX = "/"
+
+
+class Storage(object):
+    @staticmethod
+    def download(uri, local_dir):
+        if uri.startswith(_GCS_PREFIX):
+            Storage._download_gcs(uri, local_dir)
+        elif uri.startswith(_S3_PREFIX):
+            Storage._download_s3(uri, local_dir)
+        elif uri.startswith(_LOCAL_PREFIX):
```

Comment:
what does local here refer to? and are we accounting for storage like PV etc?

Reply:
Mostly for developer workflow, or if you happen to mount a PV. The PV should be able to be read directly using this.